test-framework/junit5/src/main/java/io/quarkus/test/junit/IntegrationTestUtil.java

```diff
@@ -498,7 +498,7 @@ private static Path determineBuildOutputDirectory(final URL url) {
             if (url.getPath().endsWith("test-classes/")) {
                 // we have the maven test classes dir
                 return toPath(url).getParent();
-            } else if (url.getPath().endsWith("test/")) {
+            } else if (url.getPath().endsWith("test/") || url.getPath().endsWith("intTest/")) {
```

Comment:
We should check whether what we are doing if `url.getPath().endsWith("-tests.jar")` will work for all the other cases. That could be a separate PR though.